.golangci.yml

```diff
@@ -0,0 +1,41 @@
+run:
+  # timeout for analysis, e.g. 30s, 5m, default is 1m
+  timeout: 5m
+
+  # Skip linting _test.go files
+  tests: false
+
+  # Enables skipping of directories:
+  # vendor$, third_party$, testdata$, examples$, Godeps$, builtin$
+  skip-dirs-use-default: true
+
+  # Allow multiple parallel golangci-lint instances running.
+  # If false (default) - golangci-lint acquires file lock on start.
+  allow-parallel-runners: true
+
+linters:
+  enable:
+    - bodyclose
+    - deadcode
+    - errcheck
+    - gocyclo
+    - gofmt
+    - gosimple
+    - govet
+    - ineffassign
+    - lll
+    - staticcheck
+    - structcheck
+    - typecheck
+    - unused
+    - varcheck
+
+linters-settings:
+  gocyclo:
+    min-complexity: 20 # default is 30.
+
```

Comment:
I'm not sure how to add this, but there's also a setting for `goimports` (see [1]) for sorting local imports (I'm sure @merlin would appreciate this).
https://github.com/mendersoftware/devicemonitor/blob/master/.golangci.yml#L38

Reply:
I like it. Let me think a bit how can we use this in a generic way...

Reply:
```-local=github.com/mendersoftware```
Would be nice at least (for `goimports`).

Reply:
@oleorhagen Yep :+1:

Reply:
@alfrunes I sure would! :)